h/static/scripts/widget-controller.js

```diff
@@ -322,6 +370,8 @@ module.exports = function WidgetController(
   };
 
   $scope.isLoading = isLoading;
+  $scope.tabAnnotations = annotationUI.TAB_ANNOTATIONS;
```

Comment:
Ah, I wondered where `tabAnnotations` was coming from. IMO it would be better to put these constants in a place that they can just be directly imported from the component and tests that use them.